src/platform/bouffalolab/BL602/BUILD.gn

```diff
@@ -30,34 +32,43 @@ static_library("BL602") {
   ]
 
   common_sources = [
```

Comment:
why are common sources separated from sources when we do a `source += common_sources` in the end anyway?

Reply:
remove common_sources